prover/prover_fri_gateway/src/proof_gen_data_fetcher.rs

```diff
@@ -14,19 +14,7 @@ impl PeriodicApiStruct {
             .await
             .expect("Failed to save proof generation data to GCS");
         let mut connection = self.pool.connection().await.unwrap();
-        connection
-            .fri_protocol_versions_dal()
-            .save_prover_protocol_version(data.protocol_version_id, data.l1_verifier_config)
-            .await;
-        connection
-            .fri_witness_generator_dal()
-            .save_witness_inputs(
-                data.l1_batch_number,
-                &blob_url,
-                data.protocol_version_id,
-                data.eip_4844_blobs,
-            )
-            .await;
+        todo!() // Will be fixed in prover PR.
```

Comment:
is it okay to merge the PR in the current state?

Reply:
Not okay, the plan is to merge server and prover changes to main in a single PR

Reply:
can we merge the prover PR to this branch, then?

Reply:
Do you mean now? I would prefer to do it at the end when tests pass there and all review comment are resolved. Then we can merge and do final round of review